scripts/run.sh

```diff
@@ -41,7 +41,7 @@ if [ -n "${CAMPAIGNS:-}" ] ; then
 fi
 
 # dispatch job
-${CAMPAIGNS:-/opt/campaigns/${TYPE}}/scripts/run.sh ${INPUT_FILE} ${EVENTS_PER_TASK} ${TASK}
+${CAMPAIGNS:-/opt/campaigns/hepmc3}/scripts/run.sh ${EXT} ${INPUT_FILE} ${EVENTS_PER_TASK} ${TASK}
```

Comment:
Also reverse INPUT_FILE and EXT, and rename INPUT_FILE_BASENAME or so to make clear it excludes extension.